### ISSUE TITLE
Machine stuck on echo 3 > drop_caches

### ISSUE THREAD

**Issue by airstand** [mpalmer/lvmsync#49]
Hello,

If you are using KSM enabled on Host node for KVM VMs .. there is a problem dropping caches.

The best way to do that is :

        #File.open("/proc/sys/vm/drop_caches", 'w') { |fd| fd.print "3" }

        system("sync; /sbin/sysctl vm.drop_caches=1");

Regards,
Spas

**Comment by LukasdeBoer** [comment]
Yes, I had the same issue on a Proxmox machine - this fixed it. Thanks!
